Makefile

```diff
@@ -271,3 +271,17 @@ vendor:
 	cd pkg/hardwareutils; go mod vendor
 	cd hack/tools; go mod vendor
 	go mod vendor
+
```

Comment:
this change also does not apply to us

Reply:
I'm not sure we should diverge from the upstream Makefile. But we should make sure the github workflow will not run.